elkm1_lib/keypads.py

```diff
@@ -61,6 +74,13 @@ def _kc_handler(self, keypad: int, key: int) -> None:
             name = ""
         self.elements[keypad].setattr("last_keypress", (name, key), True)
 
+    def _kf_handler(self, keypad: int, key: str, chime_mode: list[int]):
+        try:
+            name = FunctionKeys(key).name
+        except ValueError:
+            name = ""
+        self.elements[keypad].setattr("last_function_key", (name, key), True)
```

Comment:
I think you need to do something similar to the `_kc_handler` where the first line is setting the last_keypress to None. The notifier works on deltas and with key presses the user can press the same key twice in a row and there would only be a single notification.

Reply:
I wondered about that, change made.